src/RiakClient/Models/Index/SecondaryIndex.cs

```diff
@@ -30,7 +30,6 @@ namespace RiakClient.Models.Index
     /// </summary>
     /// <typeparam name="TClass">The concrete type of the implementing class.</typeparam>
     /// <typeparam name="TIndex">The type of the index (<see cref="BigInteger"/> or <see cref="string"/>).</typeparam>
-    [ComVisible(false)]
```

Comment:
Also what did we change that unclogged a bunch of these not com visible classes?